books/free-programming-books-subjects.md

```diff
@@ -223,6 +223,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 
 ### Containers
 
+* [CI/CD for Monorepos: Effectively building, testing, and deploying code with monorepos](https://semaphoreci.com/resources/monorepo-cicd) - Pablo Tomas Fernandez Zavalia, Marko Anastasov, SemaphoreCI (PDF, EPUB, Kindle, [Markdown, TeX](https://github.com/semaphoreci/book-monorepo-cicd))
```

Comment:
The github link is right there on the first link. It's not our job to deconstruct their website.

Reply:
Ok like in #6465